query-engine/js-connectors/src/proxy.rs

```diff
@@ -153,12 +154,11 @@ pub enum ColumnType {
 #[derive(Debug)]
 pub struct Query {
     pub sql: String,
-    pub args: Vec<serde_json::Value>,
+    pub args: Vec<JSArg>,
 }
+
 /// Handle data-type conversion from a JSON value to a Quaint value.
 /// This is used for most data types, except those that require connector-specific handling, e.g., `ColumnType::Boolean`.
-/// In the future, after https://github.com/prisma/team-orm/issues/257, every connector-specific handling should be moved
-/// out of Rust and into TypeScript.
 fn js_value_to_quaint(
```

Comment:
I wonder if this function would also look more in place in `conversion` module.

Reply:
Potentially, but we'd need moving e.g. `ColumnType` as well to avoid circular dependencies. If we want to do it, let's do it in a follow-up PR (so that git conflicts are less likely, and so that the diff isn't larger than it already is)